src/scripts/main.js

```diff
@@ -1,3 +1,230 @@
 'use strict';
 
-// write your code here
+const button = document.querySelector('.button');
+const message = document.querySelector('.message-start');
+const lose = document.querySelector('.message-lose');
+const win = document.querySelector('.message-win');
+const rows = document.querySelectorAll('tr');
+const allCells = document.querySelectorAll('td');
+let buttonClicked = false;
+
+button.addEventListener('click', () => {
```

Comment:
buttonClicked is set to true at the start of the function, so the if (!buttonClicked) condition will never be true. You should review the logic here.